lighthouse-core/computed/metrics/lantern-first-contentful-paint.js

```diff
@@ -24,52 +26,109 @@ class LanternFirstContentfulPaint extends LanternMetric {
   }
 
   /**
-   * @param {Node} dependencyGraph
-   * @param {LH.Artifacts.TraceOfTab} traceOfTab
-   * @return {Node}
+   *
+   * @param {Node} graph
+   * @param {number} fcpTs
+   * @param {function(NetworkNode):boolean} networkFilter
+   * @param {function(CPUNode):boolean=} cpuFilter
+   * @return {{possiblyRenderBlockingScriptUrls: Set<string>, actuallyRenderBlockingScriptUrls: Set<string>, blockingCpuNodeIds: Set<string>}}
```

Comment:
`actually` reads like `possibly` blocking nodes really weren't. `definitelyRenderBlockingScriptUrls`? :)

Reply:
haha good point sounds good 👍

Reply:
well, on further reflection `definitely` feels a tad too strong :)

we don't not with 100% certainty, but it's the delta between the two is `definitelyNotRenderBlocking`, maybe I just return that directly